### DISCUSSION TITLE
Denoise SD3.5, Flux  vs  SDXL

### DISCUSSION THREAD

**Discussion by Creative-comfyUI** [comfyanonymous/ComfyUI#5833] [General]
It seems that since the arrival of FLUX and SD3.5, Denoise has much less influence or none at all. Is it because SD3.5 and Flux no longer use UNET?

**Comment by ltdrdata** [comment]
It is likely due to the difference in model capabilities.

Previous generations of models used a mechanism where they wandered through the noise space during the denoising process, gradually arriving at the final result. In contrast, recent models employ a technique that denoises with a direct and straightforward directional approach, avoiding unnecessary wandering.

The advantage of this is that it produces more accurate results, while the drawback could be seen as generating outcomes that lack creativity.

**Reply by clayne** [reply]
I also independently discovered this. The denoise setting with 3.5 (and perhaps Flux) is incredibly sensitive as compared to XL. Whereas a value of 0.2 with XL would barely change much, with 3.5 it significantly changes the output, greatly reducing the useful range of the denoise setting, atleast at lower values.
